docs/DeveloperGuide.md

```diff
@@ -427,36 +476,58 @@ testers are expected to do more *exploratory* testing.
 
    1. Resize the window to an optimum size. Move the window to a different location. Close the window.
 
-   1. Re-launch the app by double-clicking the jar file.<br>
+   2. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
-
 ### Deleting a person
 
-1. Deleting a person while all persons are being shown
+1. Deleting a person while persons are being shown
 
-   1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
+   1. Prerequisites: List all persons using the `list` command. Multiple clients with the same name exist.
 
-   1. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
+   2. Test case: `delete id/1`<br>
+      Expected: First client is deleted from the list. Details of the deleted client shown in the status message.
 
-   1. Test case: `delete 0`<br>
-      Expected: No person is deleted. Error details shown in the status message. Status bar remains the same.
+   3. Test case: `delete id/0`<br>
+      Expected: No client is deleted. Error details shown in the status message.
 
-   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
+   4. Test case `delete n/John Doe` <br>
+      Expected: Client with name `John Doe` is deleted. Details of the deleted client shown in the status message. <br>
+      If multiple `John Doe`s exist, list of all `John Doe`s is displayed. <br>
+      User prompted to delete by index in status message.
+   
+   5. Other incorrect delete commands to try: `delete`, `delete p/x` (incorrect prefixes), `delete id/y`, `...` (where y is larger than the list size)<br>
       Expected: Similar to previous.
 
-1. _{ more test cases …​ }_
-
 ### Saving data
 
-1. Dealing with missing/corrupted data files
-
-   1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
-
-1. _{ more test cases …​ }_
+1. Dealing with missing or corrupted data files
+
+   1. `preferences.json`
+      1. Move `preferences.json` to another folder.
+      2. Relaunch the app.
+      
+      Expected: Application launches with the default window size.
+
+   2.  `config.json`
+       1. Move `config.json` to another folder.
+       2. Relaunch the app.
+       
+       Expected: Application launches, config file is re-created with default configuration.
+
+   3.  `addressbook.json`
+       1. Corrupted:
+          1. Manually modify `addressbook.json`
+          2. Relaunch the app 
+          
+          Expected: Application fails to read from the file and loads without any data.
+       
+       2. Missing:
+          1. Move `data` folder to another directory.
+          2. Relaunch the app. 
+          
+          Expected: Application loads with last saved data.
```

Comment:
Is it last saved data? I thought if addressbook.json is missing its loaded with the sample address book

Reply:
yes thanks for pointing that out!